src/app/results/results.component.html

```diff
@@ -67,6 +68,9 @@
 
   <app-infobox [hidden]="hidefooter" class="infobox col-md-4" *ngIf="Display('all')"></app-infobox>
   <!-- END -->
+  <div id="map">
+    <script src="https://unpkg.com/leaflet@1.1.0/dist/leaflet.js"></script>
```

Comment:
Leaflet map is called inside a div tag with `id="map"` and renders map if script is inside div tag. But it looks like leaflet is failing tests with typescript 2.0. We need to find an alternative to pass the tests.